content/blog/bitigchi/2020-09-11-system-sound-contest.md

```diff
@@ -0,0 +1,64 @@
++++
+type = "blog"
+title = "Contest for System Sounds"
+author = "bitigchi"
+date = "2020-09-06 13:00:00+03:00"
+tags = ["sound", "contest"]
++++
+
+It wouldn't be wrong to say that, Haiku is a unique operating system. It takes its roots from BeOS, the OS everyone envied back in the day, and Haiku follows its footsteps in the most loyal way possible. What could be better than complementing this unique operating system with a unique set of system sounds?
```

Comment:
"in the most loyal way possible."
Isn't that a bit over the top, even for this extremely enthusiastic 1st paragraph? :)

Reply:
Updated.